// @lmc-eu/spirit-web-react/components/Tag/Tag.stories.js
var __assign = (this && this.__assign) || function () {
    __assign = Object.assign || function(t) {
        for (var s, i = 1, n = arguments.length; i < n; i++) {
            s = arguments[i];
            for (var p in s) if (Object.prototype.hasOwnProperty.call(s, p))
                t[p] = s[p];
        }
        return t;
    };
    return __assign.apply(this, arguments);
};
import React from 'react';
import Tag from './Tag';
export default {
    title: 'Components/Tag',
    argTypes: {
        color: {
            options: ['default', 'informative', 'success', 'warning', 'danger'],
            control: { type: 'select' },
        },
        theme: {
            options: ['light', 'dark'],
            control: { type: 'select' },
        },
        children: {
            control: 'text',
        },
    },
};
var Template = function (args) { return React.createElement(Tag, __assign({}, args)); };
export var ExampleTag = Template.bind({});
ExampleTag.args = {
    color: 'default',
    theme: 'dark',
    children: 'Tag example',
};
